scrape.js

```diff
@@ -16,6 +16,13 @@ const linkedinLogin = async (username, password, page) => {
 
   // Wait for page load
   await page.waitForNavigation();
```

Comment:
This line isn't needed anymore and would be a problem for users without 2FA as the page doesn't navigate.